modules/ossm-deploy-application-workloads-per-user-group.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+// install/ossm-using-multiple-control-planes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-per-user-group_{context}"]
+= Deploy application workloads per user group
+:context: ossm-deploy-application-workloads-per-user-group
+
+This procedure demonstrates how to deploy application workloads in separate namespaces based on user groups.
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Label the namespace with the respective control plane:
```

Comment:
```suggestion
. Add the `usergroup=usergroup-1` label so that the namespace is discovered by the first control plane and the `istio.io/rev=mesh-1` label so that side car injection is enabled in the namespace by default. Note that `mesh-1` corresponds to the revision name, which can be confirmed with the command `oc get istiorevisions`:
```